transport/transport-backend-cct/src/main/java/com/google/android/datatransport/cct/CctTransportBackend.java

```diff
@@ -334,6 +336,10 @@ private HttpResponse doSend(HttpRequest request) throws IOException {
     connection.setRequestProperty(CONTENT_TYPE_HEADER_KEY, JSON_CONTENT_TYPE);
     connection.setRequestProperty(ACCEPT_ENCODING_HEADER_KEY, GZIP_CONTENT_ENCODING);
 
+    if(request.pseudoId != null) {
```

Comment:
since we are using the full name in the rest of the code, we can do so here and elsewhere`pseudonymousId`